faq.md

```diff
@@ -12,18 +12,13 @@ permalink: /faq/
 
 Yes, <a href="download/">choose a download</a>.
 
-Don't be an idiot and download a binary claiming to be of this project. In fact,
-don't be an idiot and download *any* binary claiming to be an Xbox 360 or PS3
-emulator from *any* source, especially not YouTube videos and shady websites.
-Come on people. Jeez.
-
 ## Xenia crashes when I run it!
 
+Make sure your PC meets the system requirements of xenia.
 This project is in early development phases and is not meant to be used by those
 who aren't comfortable debugging the code. Issues filed with 'it's broken' are
 generally unhelpful, and in most cases the team knows what's broken (the
-assumption is everything). If it doesn't work for you and you cannot figure out
```

Comment:
I don't think this needed to be removed?

Reply:
I think that'd be fine, but changing "support" to something more like "community-provided support" might be a good idea.